### PR TITLE
add foreground run mode in dev run script

### PR DESCRIPTION
**Pull request by sbaogang**
Per user request, add foreground run mode in the dev run scripts.

@jinruh @yujunglo @nickdechant

### CODE REVIEW

File: deploy-board/run.sh [hunk 1]
```diff
@@ -12,9 +17,15 @@ function server_start {
...
-    python $ROOT_DIR/manage.py runserver 0.0.0.0:8888 >& /tmp/deploy_board/access.log &
 
-    echo "Deployboard started. Check log at /tmp/deploy_board/service.log"
+    if [ "$1" == "FOREGROUND" ]
+    then
+        echo "Deployboard started. Check log at /tmp/deploy_board/service.log"
+        python $ROOT_DIR/manage.py runserver 0.0.0.0:8888 >& /tmp/deploy_board/access.log
+    else
+        python $ROOT_DIR/manage.py runserver 0.0.0.0:8888 >& /tmp/deploy_board/access.log &
+        echo "Deployboard started. Check log at /tmp/deploy_board/service.log"
```
Comment (jinruh) [inline deploy-board/run.sh]: do you also want to write a pid file somewhere so the daemon watcher can watch it? (like monitd)

Comment (sbaogang) [inline deploy-board/run.sh]: good point, will add that.

**Comment by sbaogang** [comment]
Make the run scripts more production worthy.

File: deploy-board/run.sh [hunk 2]
```diff
@@ -1,32 +1,71 @@
...
+    mkdir -p ${LOG_DIR}
 
-    echo "Deployboard started. Check log at /tmp/deploy_board/service.log"
+    export PYTHONPATH=${ROOT_DIR}
+    if [ "$1" == "FOREGROUND" ]
+    then
+        python ${MANAGE_FILE} runserver ${ADDR_PORT}
+    else
+        python ${MANAGE_FILE} runserver ${ADDR_PORT} >& ${LOG_DIR}/access.log &
+        #echo $BASHPID > ${PID_FILE}
```
Comment (yujunglo) [inline deploy-board/run.sh]: just directly remove this

File: deploy-board/run.sh [hunk 3]
```diff
@@ -1,32 +1,71 @@
...
+        echo $! > ${PID_FILE}
+        echo "Deployboard started."
+    fi
 }
 
 function server_stop {
-    echo "Stopping Deployboard..."
-    pkill -f "deploy-board/manage.py runserver"
+    #pkill -f "deploy-board/manage.py runserver"
```
Comment (yujunglo) [inline deploy-board/run.sh]: and this

**Comment by yujunglo** [comment]
looks good to me. Thanks.

**Pull request merged**